src/stem-tracker.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+var Stem = require("./stem");
+
+/**
+ * Stem Tracking Utility.
+ *
+ * Handles the stems for a given editor.
+ */
+module.exports = function StemTracker(editor, onClick) {
+  var me = this;
+  var previousStems = [];
```

Comment:
previousStems can be a local variable in the only function that uses it.  It's not part of any ongoing state of the StemTracker.